Library/Formula/qt.rb

```diff
@@ -25,6 +25,9 @@ class Qt < Formula
     sha256 "c8a0fa819c8012a7cb70e902abb7133fc05235881ce230235d93719c47650c4e"
   end
 
+  # Make `HOMEBREW_PREFIX/lib/qt4/plugins` an additional plugin search path.
+  patch :DATA
```

Comment:
All the alternatives I could come up with, that do not involve patching the source, are much more complicated or cumbersome and more difficult to get right. Would patching the binaries after they have been built a more acceptable solution? (I'm asking because we seem to be less shy about deleting/moving stuff and creating symlinks all over the place after everything has been built, even if upstream never intended it to be like this. And there isn't even a reference to an upstream issue.)